slither/printers/guidance/echidna.py

```diff
@@ -390,40 +391,43 @@ def output(self, filename: str) -> Output:  # pylint: disable=too-many-locals
             _filename(string)
         """
 
-        payable = _extract_payable(self.slither)
+        filter = r"mock(s)?|test(s)?"
+        contracts = [c for c in self.slither.contracts if not re.search(filter, c.file_scope.filename.absolute, re.IGNORECASE)]
```

Comment:
Is there a reason not to use `is_test_contract` https://github.com/crytic/slither/blob/43888ef284a26e98c47faf103732c3f55343101b/slither/utils/tests_pattern.py#L36